cni-plugin/tests/calico_cni_k8s_test.go

```diff
@@ -88,7 +88,8 @@ func ensurePodDeleted(clientset *kubernetes.Clientset, ns string, podName string
 		podName,
 		metav1.DeleteOptions{
 			PropagationPolicy:  &fg,
-			GracePeriodSeconds: &zero})
+			GracePeriodSeconds: &zero,
```

Comment:
whoops looks like my formatter "fixed" a bunch of improperly formatted lines :sweat_smile: